packages/lexical-website-new/src/css/custom.css

```
@@ -40,3 +40,7 @@ html[data-theme='dark'] .docusaurus-highlight-code-line {
 .footer__links {
   margin-bottom: 3rem;
 }
+
+.button--outline:hover {
```

Can we do something like the below if this is only conditional on it being in dark mode? 

```suggestion
html[data-theme='dark'] .button--outline:hover {
```